From e206600f606b8572cd649c19c7324552f50c3a24 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Thu, 14 Nov 2013 18:21:49 +0000
Subject: [PATCH 127/462] Mantis 29552: device documentation

ReleaseNotes.schema.json; V1.0.0
initial version for structured release notes using JSON schema file

added all authors for SelfServiceBuild, SelfServiceCommon, SelfServicePlatform

git-svn-id: svn://localhost/SelfServiceBuild/trunk@189 2eb42a87-3827-0410-b767-dc15055982e4
---
 .../Build/releasenotes/src/ReleaseNotes.schema.json             | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SelfServiceBuild/Build/releasenotes/src/ReleaseNotes.schema.json b/SelfServiceBuild/Build/releasenotes/src/ReleaseNotes.schema.json
index ea92e45..5a9b7d5 100644
--- a/SelfServiceBuild/Build/releasenotes/src/ReleaseNotes.schema.json
+++ b/SelfServiceBuild/Build/releasenotes/src/ReleaseNotes.schema.json
@@ -328,7 +328,7 @@
    "type": "string"
   },
   "Author": {
-   "enum": ["GlT", "JuV", "KhJ", "JSchyr", "SteT"],
+   "enum": ["GlT", "JuV", "KhJ", "JSchyr", "SteT", "SchiN", "KpS", "GeA", "PtM"],
    "type": "string"
   },
   "System": {
-- 
2.41.0.windows.1

